core/src/main/java/com/linecorp/armeria/common/CommonPools.java

```diff
@@ -52,7 +52,10 @@
 
     /**
      * Returns the common blocking task {@link Executor} which is used when
-     * {@link ServerBuilder#blockingTaskExecutor(Executor)} is not specified.
+     * {@link ServerBuilder#blockingTaskExecutor(Executor)} is not specified, or
+     * {@link com.linecorp.armeria.client.retrofit2.ArmeriaRetrofitBuilder#streaming(boolean)} is true and
+     * {@link com.linecorp.armeria.client.retrofit2.ArmeriaRetrofitBuilder#callbackExecutor(Executor)}
```

Comment:
Doesn't this make `./gradlew :core:javadoc` fail, because `ArmeriaRetrofit` is not available? How about simplifying this way:

```
Returns the default common blocking task {@link Executor} which is used for
running a potentially long-running tasks which may block I/O threads.
```